open_spiel/python/mfg/games/crowd_modelling.py

```diff
@@ -72,8 +72,14 @@ class MFGCrowdModellingGame(pyspiel.Game):
   """
 
   # pylint:disable=dangerous-default-value
-  def __init__(self, params: Mapping[str, Any] = _DEFAULT_PARAMS):
-    super().__init__(_GAME_TYPE, _GAME_INFO, params)
+  def __init__(
+    self, params: Mapping[str, Any] = None, game_info: Mapping[str, Any] = None
+  ):
+    if game_info is None:
+      game_info = _GAME_INFO
```

Comment:
I think that it would be better to dynamically build here the game_info that you pass to the super class, based on the parameters. You just need to fill max_chance_outcomes, max_game_length correctly based on the parameters.

Reply:
Thanks. Will do.